exchanges/poloniex/poloniex_futures.go

```diff
@@ -0,0 +1,769 @@
+package poloniex
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"slices"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchange/order/limits"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/kline"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/margin"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+)
+
+var (
+	errPositionModeInvalid     = errors.New("invalid position mode")
+	errMarginAdjustTypeMissing = errors.New("margin adjust type invalid")
```

Comment:
🐒 Optional nitpick:
Consistency
```suggestion
	errInvalidPositionMode     = errors.New("invalid position mode")
	errInvalidMarginAdjustType = errors.New("Invalid margin adjust type")
```